src/Client.php

```diff
@@ -102,14 +85,16 @@ public function getOptions(): Options
      */
     public function captureMessage(string $message, ?Severity $level = null, ?Scope $scope = null): ?string
     {
-        $payload['message'] = $message;
-        $payload['level'] = $level;
+        $payload = [
+            'message' => $message,
+            'level' => $level,
+        ];
 
-        if ($this->getOptions()->shouldAttachStacktrace()) {
-            $payload['stacktrace'] = Stacktrace::createFromBacktrace($this->getOptions(), $this->serializer, $this->representationSerializer, \debug_backtrace(DEBUG_BACKTRACE_IGNORE_ARGS), __FILE__, __LINE__);
+        if ($event = $this->prepareEvent($payload, $scope, $this->options->shouldAttachStacktrace())) {
+            return $this->send($event);
```

Comment:
Since the `send` function is just a small method that just calls in turn the transport we can inline its code here